### ISSUE TITLE
[1.8.1] Meteor legacy builds don't work for node_modules

### ISSUE THREAD

**Issue by sebakerckhof** [meteor/meteor#10563]
It's the first time I need to create a Meteor app which should support older browser versions.
I thought I'd finally be able to profit from the legacy/modern split builds. But it turns out my imported node_modules aren't being transpiled and therefore I get syntax errors in older browsers.

Minimal reproduction: https://github.com/sebakerckhof/meteorlegacy
This is a default meteor boilerplate app, in which I just added 1 npm module: 'pify' which I import on the client side. The application then breaks in older browsers, because the pify module (part of the packages/modules.js) is not transpiled and contains const declarations, arrow functions etc.

Can be verified e.g. with an ie9 vm from https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/

When I symlink the package into the /imports folder, it does get transpiled, but I thought this was no longer necessary since 1.8...

**Comment by benjamn** [comment]
The symlinking trick is still necessary in 1.8.1, but perhaps it's time to start compiling `node_modules` more aggressively (by default) for the `web.browser.legacy` bundle.

**Comment by sebakerckhof** [comment]
Aah, ok. I must have misunderstood. I thought because of the lazy compilation the imports symlink was no longer necessary. Then again, doing it by default might increase rebuild times for legacy substantially, but maybe this isn't much of an issue once https://github.com/meteor/meteor/pull/10427 lands.

**Comment by CaptainN** [comment]
I'm getting an error with material-ui 4.01. which ships a esm folder, and has its package.json's module field set up to use the content from within esm (esm is pre-compiled, except the import/export statements, which is what fails for me).

It'd also be nice to be able to tell it to use the es folder instead, since those are non-precompiled, and we may be able to get some modern bundle benefits by starting with those modules, instead of esm modules.

```
W20190530-16:33:39.433(-4)? (STDERR) (node:26476) UnhandledPromiseRejectionWarning: C:\Users\Kevin\repos\pixstori\node_modules\@material-ui\core\esm\Card\index.js:1
W20190530-16:33:39.434(-4)? (STDERR) (function (exports, require, module, __filename, __dirname) { export { default } from './Card';
```

If this is a separate issue, I'll move it to its own ticket.

**Comment by benjamn** [comment]
Closing thanks to the workaround implemented in https://github.com/meteor/meteor/pull/10603. I wish legacy builds could just automatically recompile `node_modules` code without any additional configuration, but that approach leads to severe performance problems, e.g. https://github.com/meteor/meteor/issues/10598.
